reactos/drivers/usb/usbhub_new/debug.c

```diff
@@ -45,7 +45,7 @@ USBPORT_DumpingConfiguration(IN PUSB_CONFIGURATION_DESCRIPTOR ConfigDescriptor)
 
     Descriptor = (PUSB_COMMON_DESCRIPTOR)ConfigDescriptor;
 
-    do
+    while (Descriptor->bLength)
```

Comment:
I was expecting
```
while ((ULONG_PTR)Descriptor) < (ULONG_PTR)ConfigDescriptor + ConfigDescriptor->wTotalLength)
```

But you can keep it the way you have it for now (unless you find a bug with it) -- I'll let you know if I see any indication that the current version is broken.